### DISCUSSION TITLE
Use variable in googleSheet Update

### DISCUSSION THREAD

**Discussion by amaillet2024** [AutomaApp/automa#1687] [Q&A]
Hello, 
I need to update 2 selected cell in a google sheet. I don't want to create a table for 2 fixed data in my workflow.
(I change the edited line in a loop and need to update row by row)

When I use fixed data it works fine.
For example : `[["Publié","01/04/2424"]]`

but if I change to `[["Publié",{{variable@today}}]] `
it doesn't work, I have the error : `[body] "values" must be an array On the Google Sheets block `

I tried to set a var to an array, for exemple : myVar => `[["Publié",{{variable@today}}]] ` and use `{{variables@myVar}}` in the googlesheet block
also by using JS block...

But it doesn't help

can someone help me ?

regards, 
Annabelle

**Comment by juliocfc92** [comment]
[["Publié","{{variables.today}}"]]

**Reply by juliocfc92** [reply]
![image](https://github.com/user-attachments/assets/949396b6-7631-487f-8358-27fc05cd455f)
